.golangci.yml

```diff
@@ -0,0 +1,29 @@
+linters:
+  enable:
+    - gofmt
+    - bodyclose
+    - errcheck
+    - goimports
+    - errorlint
+    - exportloopref
+    - gosimple
+    - govet
+    - ineffassign
+    - misspell
+    - rowserrcheck
+    - sqlclosecheck
+    - staticcheck
+    - typecheck
+    - unparam
+linters-settings:
+  goimports:
+    local-prefixes: github.com/tensorchord/envd-server/
+run:
+  timeout: "1m"
+  issue-exit-code: 1
+  tests: true
+  skip-dirs-use-default: true
+  allow-parallel-runners: false
+  skip-dirs:
+    - client
+    - pkg/errdefs
```

Comment:
May I know why these two dirs are skipped?

Reply:
These files are copied from docker/docker and it is not used in the envd-server.

It is the golang SDK used in envd CLI. Thus there are many unused errors. And I cannot disable unused check because of unknown reasons.